res/database/Default/database_ads.xml

```diff
@@ -1712,7 +1714,7 @@ Available in any well-stocked drugstore.</en>
 			</title>
 			<description>
 				<de>Premium Honigprodukte von Ihrer Lieblings-Imkerei Hummeldoof. Wir gewähren unseren Bienen ein gutes, motivierendes Arbeitsumfeld. Feierabendbier inklusive!</de>
-				<en>Premium honey products from your favorite apiary Dumblebee. We offer our bees a good, motivating work environment. Including after-work pints!</en>
+				<en>Premium honey products from your favorite apiary Dumblebee. We provide our bees a good, motivating work environment. Including after-work pints!</en>
```

Comment:
check provide ( provide x with; provide y to/for x)

Reply:
it's fine. https://www.google.com/search?q="we+provide+our"

Reply:
"We provider our bees with a good..."

Reply:
it's common with our without the "with", but perhaps it's clearer with the "with" `:-]`